pkg/component/controller/cplb/cplb_label_linux.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2025 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cplb
+
+import (
+	"fmt"
+	"net"
+	"unsafe"
+
+	"github.com/vishvananda/netlink/nl"
+	"golang.org/x/sys/unix"
+)
+
+// maybeSetAddrLabels sets the labels for the vips to vrrp.AddressLabel
+// so that the real IP is preferred.
+func (k *Keepalived) maybeSetAddrLabels() error {
```

Comment:
What about moving this method into `cplb_linux.go` and rename it to `setVirtualIPAddressLabels`?

Reply:
That's a better name, done.